src/src/sl_rcp_gp_interface.c

```diff
@@ -0,0 +1,235 @@
+/*******************************************************************************
```

Comment:
Is this the correct license header for stuff committed to the openthread/ot-efr32 GitHub repo?

Reply:
Good catch. Lemme fix that quick